### PR TITLE
scm/github: filter issues that belongs to PR diff

### PR DESCRIPTION
**Pull request by herlon214**
This should fix the issue https://github.com/herlon214/sonarqube-pr-issues/issues/8

### CODE REVIEW

**Comment by adenisonafifi** [comment]
Thank you for following up on this so quickly!

**Comment by herlon214** [comment]
@adenisonafifi no problem, I was adding tests to be sure it won't break. Worth a star ⭐  haha
I'm gonna release a new version

**Comment by herlon214** [comment]
@adenisonafifi v0.5.0 released https://github.com/herlon214/sonarqube-pr-issues/releases/tag/v0.5.0

File: pkg/scm/github.go [hunk 1]
```diff
@@ -56,22 +57,72 @@ func (g *Github) PublishIssuesReviewFor(ctx context.Context, issues []sonarqube.
...
 		}
-		comments = append(comments, comment)
+
+	}
+
+	if len(comments) == 0 {
+		return errors.New("failed to find relevant issues")
 	}
 
 	body := fmt.Sprintf(`:wave: Hey, I added %d comments about your changes, please take a look :slightly_smiling_face:`, len(issues))
```
Comment (adenisonafifi) [inline pkg/scm/github.go:128]: @herlon214 I forgot to update the `I added %d comments about your changes` to use `len(comments)` as now the `len(issues)` >= `len(comments)`

Comment (herlon214) [reply pkg/scm/github.go:128]: Good catch

Comment (herlon214) [reply pkg/scm/github.go:128]: Fixed on v0.5.1, thanks @adenisonafifi

**Pull request merged**